frontend/app/src/people/main/body.tsx

```diff
@@ -555,43 +561,32 @@ export default function BodyComponent({ selectedWidget }) {
     // desktop mode
     return (
       <Body
-        onScroll={handleScroll}
+        onScroll={(e) => {
+          setScrollValue(e?.currentTarget?.scrollTop >= 20);
+          handleScroll(e);
+        }}
         style={{
           background: '#f0f1f3',
```

Comment:
ik you didnt change this line but would be nice to see this using the colors file too :)